### ISSUE TITLE
Please support apks

### ISSUE THREAD

**Issue by Mrothyr** [ImranR98/Obtainium#1056]
Please support apks.

New apps for android is build as apks, not apk. For example

https://github.com/fttx/barcode-to-pc-app/releases/tag/v4.5.0

So please support it.

Thank you.

**Comment by ngocanhtve** [comment]
Have you tried adding this app to Obtainium? I just added it and Obtainium installed it normally, no need to worry about apk or apks at all.

**Comment by ImranR98** [comment]
@ngocanhtve You probably have "fallback to older releases" enabled, So it's ignoring the latest release which doesn't have an APK, and using the previous one.

**Comment by ngocanhtve** [comment]
> @ngocanhtve You probably have "fallback to older releases" enabled, So it's ignoring the latest release which doesn't have an APK, and using the previous one.

You're right. I looked again and it only installed version 4.4.2 instead of the latest 4.5 with apks.

**Comment by ngocanhtve** [comment]
> @ngocanhtve You probably have "fallback to older releases" enabled, So it's ignoring the latest release which doesn't have an APK, and using the previous one.

You're right. I looked again and it only installed version 4.4.2 instead of the latest 4.5 with apks.

**Comment by Mrothyr** [comment]
Yes, it fall back to the previous one. The APKS doesn't install on my GOS, if I download it in browser. It's a zipped folder named splits with a bunch of APK and a "toc.bp". I think it's a form of "split-apk" - I can install the app with https://github.com/Aefyr/SAI.

CU

**Comment by dreamflasher** [comment]
@ImranR98 what's your take on this? Do you have any plans to support this?

**Comment by Sarulean** [comment]
The duplicate was more specific:
"Prerequisites

Describe the feature

I would like Obtainium to support installing and updating .apks files (split APK bundles).

.apks is a common distribution format used by many projects. These files usually contain multiple split APKs that can be installed as one needs as per their phone architecture, languages, etc. Currently, Obtainium only works reliably with single .apk files, which makes it impossible to install or update apps that are distributed or at least have an option as .apks.

For example, AppManager (https://github.com/MuntashirAkon/AppManager) distributes releases as .apks with .apk. Obtainium can detect new releases, but cannot install or update them as .apks. Currently users must manually download, extract, and install using another tool like SAI. This breaks the purpose of Obtainium as an updater.

Requested behavior:

Detect .apks files as valid apk and allow installing them directly or at least download them and forward the installation process to other apps like SAI. Also, allow tracking versions and updating apps installed from .apks

This would greatly improve compatibility with modern Android app distribution.

Describe alternatives you've considered (if applicable)

Well nothing other than manually downloading .apks and installing split APKs using another installer (SAI - https://github.com/Aefyr/SAI)

Additional context

Nothing much than I can add. If needed then let me know, I will try to add them."
